owncloudApp/src/main/java/com/owncloud/android/dependecyinjection/RepositoryModule.kt

```diff
@@ -21,31 +21,26 @@ package com.owncloud.android.dependecyinjection
 
 import com.owncloud.android.data.authentication.repository.OCAuthenticationRepository
 import com.owncloud.android.data.capabilities.repository.OCCapabilityRepository
-import com.owncloud.android.data.server.repository.OCServerInfoRepository
 import com.owncloud.android.data.files.repository.OCFileRepository
+import com.owncloud.android.data.server.repository.OCServerInfoRepository
 import com.owncloud.android.data.sharing.sharees.repository.OCShareeRepository
 import com.owncloud.android.data.sharing.shares.repository.OCShareRepository
 import com.owncloud.android.data.user.repository.OCUserRepository
 import com.owncloud.android.domain.authentication.AuthenticationRepository
 import com.owncloud.android.domain.capabilities.CapabilityRepository
-import com.owncloud.android.domain.server.ServerInfoRepository
 import com.owncloud.android.domain.files.FileRepository
+import com.owncloud.android.domain.server.ServerInfoRepository
 import com.owncloud.android.domain.sharing.sharees.ShareeRepository
 import com.owncloud.android.domain.sharing.shares.ShareRepository
 import com.owncloud.android.domain.user.UserRepository
 import org.koin.dsl.module
 
 val repositoryModule = module {
-    factory<CapabilityRepository> {
-        OCCapabilityRepository(
-            get(),
-            get()
-        )
-    }
-    factory<ShareRepository> { OCShareRepository(get(), get()) }
-    factory<ShareeRepository> { OCShareeRepository(get()) }
-    factory<UserRepository> { OCUserRepository(get()) }
+    factory<AuthenticationRepository> { OCAuthenticationRepository(get(), get()) }
```

Comment:
You are following alphabetical order, right? If so, have a look at https://github.com/owncloud/android/pull/2880/files#diff-fe82a8d4a657ef2a30b6a669c7d30015R45 and the line below that one